Readme.md

```diff
@@ -304,6 +304,8 @@ Unlike `end()` the `destroy()` method does not take a callback argument.
 
 ## Pooling connections
 
+Pooling is a connection manager, which handles the connections for your queries, and reconnects if needed. If you have a longer running process, you probably want to use this instead of using connection directly.
```

Comment:
Connection pooling is a very generic concept. It looks like the first sentence is just giving it another name. Perhaps instead of that, we can provide a link to https://en.wikipedia.org/wiki/Connection_pool or somewhere similar that explains the concept of connection pooling?